tests/integration/test_fim/test_files/test_max_files_per_second/test_max_files_per_second.py

```diff
@@ -42,8 +42,12 @@ def get_configuration(request):
 
 # Tests
 
-
-def test_max_files_per_second(get_configuration, configure_environment, restart_syscheckd, wait_for_fim_start):
+@pytest.mark.parametrize('inode_collision', [
+                         (False),
+                         pytest.param(True, marks=(pytest.mark.linux, pytest.mark.darwin, pytest.mark.sunos5))
+                         ])
+def test_max_files_per_second(inode_collision, get_configuration, configure_environment, restart_syscheckd,
+                              wait_for_fim_start):
```

Comment:
It should be desirable to update test documentation, including the new `inode_collision` argument and removing unused ones like `tags_to_apply.`

Reply:
Done